src/boot/mexpr.ml

```diff
@@ -523,7 +523,23 @@ let rec symbolize (env : (ident * sym) list) (t : tm) =
     -> TmUtest(fi,symbolize env t1,symbolize env t2,symbolize env tnext)
   | TmNever(_) -> t
 
-
+(* Same as symbolize, but records all toplevel definitions and returns them
```

Comment:
Function `symbolize_toplevel` should perhaps be moved to `mlang.ml` since it is more related to MLang than MExpr.